CONTRIBUTING.md

```diff
@@ -112,6 +113,27 @@ Then verify the new Rust bindings compile and pass some basic tests:
 $ cargo test -p tests_expectations
 ```
 
+## Generating graphviz dot file
```

Comment:
Also, thanks for documenting this! I think this will help new contributors a ton :)

Another cool thing might be to commit a png generated this way, and show it at the bottom of this section.